LeaderboardBackend/Controllers/AccountController.cs

```diff
@@ -51,7 +60,7 @@ [FromServices] IAccountConfirmationService confirmationService
             return r.Match<ActionResult>(
                 success => Accepted(),
                 badRole => Accepted(),
-                emailFailed => StatusCode(StatusCodes.Status500InternalServerError)
+                emailFailed => Accepted()
```

Comment:
If you're returning `Accepted()` no matter what then there's no reason to `Match()`.